sdk/translation/ai-translation-text/package.json

```
@@ -0,0 +1,117 @@
+{
+  "name": "@azure-rest/ai-translation-text",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "Microsoft Translator Service",
```

I think the description should be updated to mention it is for a client library. I am not sure if I got the service name correctly though:
```suggestion
  "description": "An isomorphic client library for the Azure Cognitive Translator Service",
```